Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -52,6 +52,38 @@ These scripts are wrapped around the incident table, so to wrap them around anot
      
         ![image](../../doc_files/closing-mirror-snow.png)
 
+## JWT Profile Authentication Configuration
+
+Grant access token to admin is not allowed.
+Customer should have non-admin user with all necessary roles(only non-admin roles) in addition to existing  role snc_platform_rest_api_access that required to make API call
+
+Prerequisites in order to support JWT
+Create a Java Key Store and upload it to the instance
+	(Accessing from the upper menu :All→System Definition→Certificates)
+(Private key will be used as integration parameter)
+Configure a JWT signing key (use keystore from above .Keep the Key Id -  it will be used as kid integration parameter) 
+(All→System OAuth→JWT Keys)
+Create a JWT provider with a JWT signing key
+(Customer required to set  in Standard Claims the same values for aud, iss and sub that will be used as integration parameters. Claim Name sub in Standard Claims has to be existing non-admin servicenow user with all necessary roles)
```

Comment:
```suggestion
1. Create a Java KeyStore and upload it to the instance. To access the Java KeyStore, click **All** > **System Definition** > **Certificates**. The Private key will be used as the integration parameter.
2. Configure a JWT signing key. Use the Java KeyStore created in the previous step. Save the Key ID -  it will be used as the kid integration parameter. To access, click **All** > **System OAuth** > **JWT Keys**.
3. Create a JWT provider with a JWT signing key. In Standard Claims, set the same values for aud, iss and sub that will be used as the integration parameters. Claim Name sub in Standard Claims has to exist as a non-admin ServiceNow user with all necessary roles. Click **All** > **System OAuth** > **JWT providers**.
```